src/transactionComputer.ts

```diff
@@ -79,5 +104,17 @@ export class TransactionComputer {
         return value && value.length ? Buffer.from(value).toString("base64") : undefined;
     }
 
-    // TODO: add missing functions wrt. specs
+    private ensureFields(transaction: ITransaction) {
+        if (transaction.sender.length !== BECH32_ADDRESS_LENGTH) {
+            throw new errors.ErrBadUsage("The `sender` field is not set");
```

Comment:
in case of both sender and receiver, the error is wrong. the field might be set, but invalid. for chain ID, the error message is ok

Reply:
changed the error message